public/index.html

```diff
@@ -5,10 +5,10 @@
     <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="zOS" />
+    <meta name="description" content="ZERO" />
     <!-- added unsafe-eval here, but we should restrict this to *just* the Olm module, rather than for all self  -->
     <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' 'unsafe-eval';" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png" />
```

Comment:
We shouldn't change this from `%PUBLIC_URL%/logo192.png`. 

logo192, which I can see that you have updated in this pull request, is what will be used for

```
<link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Revert this change and it should be what we want to use? Not sure why it has been changed to `https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png`? Maybe I'm missing something..

Reply:
Thanks, @domw30 , I'll switch back to the default logo192